src/expandableCalendar/index.tsx

```diff
@@ -29,7 +29,7 @@ enum Positions {
 }
 const SPEED = 20;
 const BOUNCINESS = 6;
-const CLOSED_HEIGHT = 120; // header + 1 week
+const CLOSED_HEIGHT = 140; // header + 1 week (was 120)
```

Comment:
It looks like the height of a week - `WEEK_HEIGHT` is 46, so why does this only increase by 20?

Reply:
I updated it because otherwise it wouldn't work without our custom theme 

<details>
<summary>120</summary>

![image](https://user-images.githubusercontent.com/25579813/132025449-cffab775-1253-47c9-950c-827d4b96b940.png)
</details>

<details>
<summary>140</summary>

![image](https://user-images.githubusercontent.com/25579813/132025555-a6b7eac2-b8eb-427a-8fb7-c7bb5d729809.png)
</details>